vignettes/bayestestR.Rmd

```diff
@@ -28,54 +28,90 @@ if (!requireNamespace("rstanarm", quietly = TRUE)) {
 }
 
 library(knitr)
-options(knitr.kable.NA = '')
-knitr::opts_chunk$set(comment=">")
-options(digits=2)
+options(knitr.kable.NA = "")
+knitr::opts_chunk$set(comment = ">")
+options(digits = 2)
 ```
 
 
 ## Why use the Bayesian Framework?
 
-The Bayesian framework for statistics is quickly gaining in popularity among scientists, associated with the general shift towards **open and honest science**. Reasons to prefer this approach are **reliability**, **accuracy** (in noisy data and small samples), the possibility of introducing **prior knowledge** into the analysis and, critically, **results intuitiveness** and their **straightforward interpretation** [@andrews2013prior; @etz2016bayesian; @kruschke2010believe; @kruschke2012time; @wagenmakers2018bayesian]. 
+The Bayesian framework for statistics is quickly gaining in popularity among scientists, associated with the general shift towards **open and honest science**. Reasons to prefer this approach are [@andrews2013prior; @etz2016bayesian; @kruschke2010believe; @kruschke2012time; @wagenmakers2018bayesian]:
 
-In general, the frequentist approach has been associated with the focus on null hypothesis testing, and the misuse of *p*-values has been shown to critically contribute to the reproducibility crisis of psychological science [@chambers2014instead; @szucs2016empirical]. There is a general agreement that the generalization of the Bayesian approach is one way of overcoming these issues [@benjamin2018redefine; @etz2016bayesian].
+- **reliability**
```

Comment:
I'd leave this first sentence as a sentence rather than bullet points, because it offers a summary sentence that one can use in their manuscripts / answers to reviewers etc. I can *sense* that it's more useful as one sentence than bullet points 

other than that, looks good (mostly only the 80 char formatting right?)